src/coreclr/jit/gentree.cpp

```diff
@@ -17689,6 +17710,12 @@ bool GenTree::isContainableHWIntrinsic() const
     {
         case NI_Vector64_get_Zero:
         case NI_Vector128_get_Zero:
+        case NI_AdvSimd_Arm64_LoadPairScalarVector64:
+        case NI_AdvSimd_Arm64_LoadPairScalarVector64NonTemporal:
+        case NI_AdvSimd_Arm64_LoadPairVector64:
+        case NI_AdvSimd_Arm64_LoadPairVector64NonTemporal:
+        case NI_AdvSimd_Arm64_LoadPairVector128:
+        case NI_AdvSimd_Arm64_LoadPairVector128NonTemporal:
```

Comment:
I'll probably see later in the review, but under what conditions are these containable?

I didn't think Arm64 really had `ins reg, [mem]` operations outside atomic instructions; particularly for `non-temporal` operations...

Reply:
Actually, I don't see where the containment handling is happening for this. I don't see any changes to lowering